CMakeLists.txt

```diff
@@ -9,6 +9,8 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 find_package(Boost REQUIRED)
 find_package(GTest REQUIRED)
 
+add_definitions(-DBOOST_DATE_TIME_POSIX_TIME_STD_CONFIG)
```

Comment:
the nanosec should added conditionally.

if(DEFINED ENV{XXX})
  add_definitions(-DBOOST_DATE_TIME_POSIX_TIME_STD_CONFIG)
endif()

Reply:
(this works only for standalone application, for CEPH, it need another definition, will not do that)

Reply:
Done